webpack-test/package.json

```diff
@@ -5,7 +5,9 @@
   "license": "MIT",
   "main": "./dist/index.d.ts",
   "scripts": {
-    "test": "node --expose-gc --max-old-space-size=4096 --experimental-vm-modules --trace-deprecation ../node_modules/jest/bin/jest.js --logHeapUsage --runInBand --bail"
+    "test": "node --expose-gc --max-old-space-size=4096 --experimental-vm-modules --trace-deprecation ../node_modules/jest/bin/jest.js --logHeapUsage --runInBand --bail",
+    "test:metric": "node --expose-gc --max-old-space-size=4096 --experimental-vm-modules --trace-deprecation ../node_modules/jest/bin/jest.js --logHeapUsage --runInBand --bail --json | node scripts/test-metric.js",
```

Comment:
This duplication looks stupid. if I reuse `test` script, and write the `test:metric` like this: 
```json
npm run test -- --json | node scripts/test-metric.json
```
the stdout will be prepended extra line: 
![image](https://user-images.githubusercontent.com/17974631/231799402-b70ae711-5909-4c1c-9671-034569f5789b.png)
which causes `JSON.parse()` error, because it is not a valid JSON anymore.
```
SyntaxError: Unexpected token > in JSON at position 1
    at JSON.parse (<anonymous>)


```